pepper-apis/src/test/java/org/broadinstitute/ddp/route/UserRegistrationRouteTest.java

```diff
@@ -839,12 +854,27 @@ public void testAccountInvitation() {
                 Auth0TenantDto auth0TenantDto = handle.attach(JdbiAuth0Tenant.class).findByDomain(auth0Domain);
                 user.set(userDao.findUserByAuth0UserId(auth0UserId, auth0TenantDto.getId()).get());
 
-                long studyId = handle.attach(JdbiUmbrellaStudy.class).findByStudyGuid(TestConstants.TEST_STUDY_GUID).getId();
+                long tempStudyId = tempStudy.getId();
                 invitation.set(handle.attach(InvitationFactory.class).createInvitation(InvitationType.AGE_UP,
-                        studyId, user.get().getId(), "test+" + System.currentTimeMillis() + "@datadonationplatform.org"));
+                        tempStudyId, user.get().getId(), "test+" + System.currentTimeMillis() + "@datadonationplatform.org"));
 
                 handle.attach(InvitationDao.class).clearDates(invitation.get().getInvitationGuid());
                 userDao.updateAuth0UserId(user.get().getGuid(), null);
+
+                UserProfileDto profile = new UserProfileDto(user.get().getId(), "Foo", "Bar", null,
+                        LocalDate.of(1953, 9, 18), null, null, null);
+
+                handle.attach(JdbiProfile.class).insert(profile);
```

Comment:
Looks like this line is causing the tests to fail? It's saying duplicate profile so I assume there's already an existing profile. This test is using the baked in test user, so I believe that user already has a profile.

Reply:
Yes, I've  been fixing this locally.  I was running the test in isolation and it was  happy, but then with all the other  tests, not so much...